docs/docs/en/guide/installation/standalone.md

```diff
@@ -2,7 +2,7 @@
 
 Standalone only for quick experience for DolphinScheduler.
 
-If you are a new hand and want to experience DolphinScheduler functions, we recommend you install follow Standalone deployment. If you want to experience more complete functions and schedule massive tasks, we recommend you install follow [pseudo-cluster deployment](pseudo-cluster.md). If you want to deploy DolphinScheduler in production, we recommend you follow [cluster deployment](cluster.md) or [Kubernetes deployment](kubernetes.md).
+If you are a newbie and want to experience DolphinScheduler functions, we recommend you install follow Standalone deployment. If you want to experience more complete functions and schedule massive tasks, we recommend you install follow [pseudo-cluster deployment](pseudo-cluster.md). If you want to deploy DolphinScheduler in production, we recommend you follow [cluster deployment](cluster.md) or [Kubernetes deployment](kubernetes.md).
```

Comment:
Actually I prefer `novice` or `green hand` instead of `newbie` here, LOL

Reply:
I do not have a strong opinion here, I see some project like spark or flink still uses `newbie` as the label in JIRA. WDYT @darkz1984